__tests__/app.test.js

```diff
@@ -56,25 +55,32 @@ describe("GET /api/articles/:article_id", () => {
   test("200 - Responds with a status code of 200", () => {
     return supertest(app).get("/api/articles/1").expect(200);
   });
-  // test("Responds with an object containing the correct article", () => {
-  //   return supertest(app)
-  //     .get("/api/articles/1")
-  //     .then(({ body }) => {
-  //       console.log(body.article);
-  //       expect(body.article).toEqual(
-  //         expect.objectContaining({
-  //           title: "Living in the shadow of a great man",
-  //           topic: "mitch",
-  //           author: "butter_bridge",
-  //           body: "I find this existence challenging",
-  //           created_at: expect.any(Date.now()),
-  //           votes: 100,
-  //           article_img_url:
-  //             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-  //         })
-  //       );
-  //     });
-  // });
+  test("Responds with an object containing the correct article", () => {
+    return supertest(app)
+      .get("/api/articles/1")
+      .then(({ body }) => {
+        expect(body.article.article_id).toEqual(1);
+        expect(body.article).toHaveProperty("title");
+        expect(body.article).toHaveProperty("topic");
+        expect(body.article).toHaveProperty("author");
+        expect(body.article).toHaveProperty("body");
+        expect(body.article).toHaveProperty("created_at");
+        expect(body.article).toHaveProperty("votes");
+        expect(body.article).toHaveProperty("article_img_url");
```

Comment:
Since this is just one article we can expect the data to look like the article we are expecting

Reply:
//PLEASE NOTE//
        //I am using toHaveProperty because the code below is not working, it says right-hand side of "instanceof" is not an object. I approached Hev with this and she suggested to use toHaveProperty just for now.